_pages/devjobs.html

```diff
@@ -30,7 +30,7 @@ <h4 class="card__title  card--jobs__title">{{ job.title }}</h4>
 
   </div>
 
-  <a href="https://epages.com/career" class="cta  cta--beyond">More developer jobs</a>
+  <a href="https://epages.com/career" class="cta  cta--beyond">Show all open positions</a>
```

Comment:
This is about the bottom below the job ads:

![Bildschirmfoto 2021-03-08 um 08 33 26](https://user-images.githubusercontent.com/28007169/110288911-f9803000-7fe8-11eb-8fd7-d9de8e1f6a70.png)

Reply:
How about "View all open positions", "View all jobs" or just "More jobs"?